llvm/lib/CodeGen/EarlyIfConversion.cpp

```diff
@@ -163,7 +163,8 @@ class SSAIfConv {
   void rewritePHIOperands();
 
 public:
-  SSAIfConv(MachineFunction &MF) {
+  /// runOnMachineFunction - Initialize per-function data structures.
+  void runOnMachineFunction(MachineFunction &MF) {
```

Comment:
This `runOnMachineFunction` is confusing. I think I'll rename it to `init` later